manager/scripts/create_page_snaps.py

```diff
@@ -42,6 +42,8 @@
     (1920, 1080),
 ]
 
+def showPath(path):
+    return "index" if (path == "" or path == "/") else path 
```

Comment:
Added this function to fix naming of files when snapping the homepage, otherwise an image named `-360x640.png` was being written.
Feel free to adjust if there's a better way for handling this :)